zulipterminal/model.py

```diff
@@ -868,6 +868,19 @@ def topics_in_stream(self, stream_id: int) -> List[str]:
 
         return list(self.index["topics"][stream_id])
 
+    def stream_email_address(self, stream_id: int) -> str:
+        """
+        Return a string containing the email address of a stream.
+        """
```

Comment:
Let's name this more explicitly with a `_fetch` prefix, to indicate this makes an API call to the server. The leading underscore will mean that we don't expect it to be called/used outside of the model.

My suggestion would be to add a separate model function which wraps the logic for determining if the data is present in the stream/subscription, and fetches it otherwise using the wrapped `.call_endpoint` method. You might consider the movement of the existing code in the popup into this new method to be a preliminary refactor; then once you have added the `_fetch` commit, you can update this other method to call it.

The docstring doesn't add much, compared to the method name and signature (ie. parameter & return value)?

Reply:
Thanks for the review! I've updated the PR accordingly. 

I'm still unsure about the docstrings though.